starter-code/basic-algorithms.js

```diff
@@ -5,3 +5,43 @@
 
 
 // Lorem ipsum generator
+
+// Names and Input
+let hacker1 = `Billy`
+console.log(`The driver's name is ${hacker1}`)
+​
+let hacker2 = `Gulfiya`
+//("Enter the navigator's name: ");
+console.log(`The navigator's name is ${hacker2}`)
+​
+​
+//Conditionals
+if (hacker1.length>hacker2.length){
+  console.log(`The Driver has the longest name, it has ${hacker1.length} characters`);
+} else if(hacker1.length<hacker2.length){
+  console.log(`Yo, navigator got the longest name, it has ${hacker2.length} characters`);
+} else {
+  console.log(`Wow, you both got equally long names, ${hacker1.length} characters!!`);
+}
+​
+let spaceName = "";
+for (i=0; i<hacker1.length; i++) {
+ spaceName = spaceName + hacker1[i] + " ";
+ console.log(spaceName.toUpperCase())
+}
+​
+let reverseName = hacker2.split("").reverse()
+ console.log(reverseName.join(""));
```

Comment:
Correct! But just want to make the point that the .join method can be put on line 33 after the .reverse. It might look a little weird on a single line, but it does mean you can console.log JUST the variable.